protocol/contracts/templegold/TempleGoldStaking.sol

```diff
@@ -267,20 +246,20 @@ contract TempleGoldStaking is ITempleGoldStaking, TempleElevatedAccess, Pausable
         
```

Comment:
I think you should remove `stakeFor()`...

The issue an auditor would find is that Bob can stakeFor() 1 wei on behalf of Alice, and that resets her clock continuously.
So she could be lcoked out for perpetuity if Bob's a real arsehole

Only other way would be for Alice to approve that Bob can stake on her behalf, first - ie she trusts Bob

Reply:
Good catch